.github/workflows/nightly.yml

```diff
@@ -91,7 +111,7 @@ jobs:
           git log --oneline --graph --no-abbrev-commit gh-pages
           echo "::endgroup::"
           echo "[command]git rebase -i --root"
-          EDITOR='sed -i "s/pick \([0-9a-z]* -nightly build-\)/drop \1/"' \
+          EDITOR="sed -i \"/^-nightly build ${GITHUB_RUN_ID}-/! s/pick/drop/\"" \
```

Comment:
> Also I think this going to look for the new run ID instead of the old one?

The `^` at the start makes it a negation. I thik you suggestion will just drop all commits?

Reply:
Ah, the `^` at the start was wrong because each line starts with `pick` not the commit message.

Reply:
The build passed, removing an old `-nightly commit-` and adding the two new ones:

https://github.com/oliver-sanders/cylc-doc/commits/gh-pages

Reply:
@MetRonnie spotted that this would drop non-nightly-build commits, yikes.

New run:

https://github.com/oliver-sanders/cylc-doc/actions/runs/4699161259

Commit list:

https://github.com/oliver-sanders/cylc-doc/commits/gh-pages